tests/e2e/stepDefinitions/createRecipeContext.js

```diff
@@ -0,0 +1,47 @@
+const { Given, When, Then } = require('@cucumber/cucumber');
+const { RecipePage } = require('../pageObjects/RecipePage')
+const { LoginPage } = require('../pageObjects/LoginPage')
+
+const assert = require('assert')
```

Comment:
playwright `expect` has useful assertions too, especially for element assertion.